.travis.yml

```diff
@@ -4,14 +4,9 @@ language: php
 
 matrix:
   include:
-    - php: 5.3
-      dist: precise
-    - php: 5.4
-    - php: 5.5
-    - php: 5.6
-    - php: 7.0
-    - php: 7.1
     - php: 7.2
+    - php: 7.3
+    - php: 7.4
     - php: nightly
   allow_failures:
     - php: nightly
```

Comment:
Do we want to test for edge?
I'd be more conservative (like the whole script) and put explicit versions.